### ISSUE TITLE
what causes 'E_Exit - Unhandled Dos 21 call 53' error?

### ISSUE THREAD

**Issue by Siddhant-Mashalkar** [joncampbell123/dosbox-x#2648]
i was trying to install Windows 1 and Windows 2 on Floppy Disks, but when Windows 1/2 tried formatting the floppy disk, 
i got this error : "E_Exit - Unhandled Dos 21 call 53"

**Comment by Wengier** [comment]
By looking at the source code, the said INT21h function is indeed not implemented at this time. Try follow the Windows install guide in DOSBox-X Wiki.

**Comment by Siddhant-Mashalkar** [comment]
yeah, same happened when i tried to format 160k floppy after booting into MSDOS 1.25
